common/assets/users-groups-roles.tmpl

```diff
@@ -0,0 +1,156 @@
+{
+  "objects": [
+  {{ if eq .objectType "user" }}
+    {
+      "id": "{{ $.input.userName }}",
+      "type": "{{ $.vars.user.object_type }}",
+      "displayName": "{{ $.input.displayName }}"
+    },
+    {
+      "id": "{{ $.input.userName }}",
+      "type": "{{ $.vars.user.identity_object_type }}",
+      "properties": {
+        "verified": true
+      }
+    },
+    {{ range $i, $element := $.input.emails }}
+      {{ if $i }},{{ end }}
+    {
+      "id": "{{ $element.value }}",
+      "type": "{{ $.vars.user.identity_object_type }}",
+      "properties":{
+        "type": "{{ $element.type }}",
+        "verified": true
+      }
+    }
+    {{ end }}
+    {{ if and ($.input.externalId) (ne $.input.externalId "") }}
```

Comment:
I'm pretty sure the second condition is redundant. `{{ if $.input.externalId }}` returns false if the string is empty. And even if it doesn't, you can use `{{ if not (empty $.input.externalId) }}` (but I don't think you need to).